portfolio/src/main/webapp/info.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+
+<html class="js-focus-visible">
+
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    
+    <title>Information</title>
+
+    <link rel="stylesheet" href="./css/bootstrap.min.css">
+    <link rel="stylesheet" type="text/css" href="./css/professionStyle.css">
+
+    
+
```

Comment:
nit: Too many new lines

("nit" generally means that the reviewer thinks this is odd and something you might have forgot while coding. Not actually a comment on the code or the design.)